packages/sentry/src/sentry-user-tracker/sentry-user-tracker.tsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
+import * as PropTypes from 'prop-types';
+import { InferPropTypes } from '../type-utils';
 import * as sentry from '../sentry';
 
-interface Props {
-  user?: sentry.User;
-}
+const userTrackerPropTypes = {
+  user: PropTypes.shape(sentry.userPropTypes),
```

Comment:
Hm... This feels weird though, what happens if the shape was different? For example, let's say user shape was
```js
id: PropTypes.string,
name: PropTypes.string,
department: PropTypes.shape({
 ...
})
```

Reply:
At this point we would need to structure the proptypes / types pretty weird right?

Reply:
Hmm I don't get your point. Can you clarify?

Reply:
AH nevermind, I understand now 🤦‍♂